research/src/pages/button.mdx

```diff
@@ -16,6 +16,20 @@ import ComponentCoverage from '../components/component-coverage'
 
 <Anatomy component="Button" />
 
+## Accessibility
+
+- Use a touch target area of `44px` by `44px`
+- For multiple buttons provided on the page that perform the same function, one of the targets should be `44px` by `44px`.
+- Buttons should get activated using `Enter` key as well as `SPACE` key.
```

Comment:
@micahgodbolt since semantic elements (button and input with type=button) already handle this behavior, this seems more specific to custom implementations (the bullet below). What are your thoughts to moving this as a bullet under "custom" below?

Reply:
good call.